pnpm-workspace.yaml

```diff
@@ -2,10 +2,11 @@
 ---
 packages:
   - "packages/*"
+  - "apps/*"
 
 # https://pnpm.io/catalogs
 catalog:
   "dts-buddy": "^0.5.0"
-  "svelte": "^5.3.0"
+  "svelte": "^5.14.4"
```

Comment:
Update this to prevent a mix of esrap 1.3 and 1.2 (though I’m not sure why this fixes it).